### PR TITLE
schedule/coordinator.go:Fix errors in the comments since the DefaultP…

### LINKED ISSUE #8173: schedule/coordinator: wrong comments for patrolRegions
we need  14 minutes since the default value of patrolRegionInterval is 100ms in the original version https://github.com/tikv/pd/pull/963

Since the DefaultPatrolRegionInterval was set to 10ms, Actually we need ` 1000000/12800/60=1.3020833333333333` minutes to iterate 1 million regions by default:
https://github.com/tikv/pd/blob/32bf572de6b55cbcf6556021f8ad0d4485247240/pkg/schedule/coordinator.go#L55

### PR DESCRIPTION
**Pull request by AndreMouche**
…atrolRegionInterval changed

### What problem does this PR solve?

Issue Number: Close #8173 

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: pkg/schedule/coordinator.go [hunk 1]
```diff
@@ -52,7 +52,8 @@ const (
 	// pushOperatorTickInterval is the interval try to push the operator.
 	pushOperatorTickInterval = 500 * time.Millisecond
 
-	patrolScanRegionLimit = 128 // It takes about 14 minutes to iterate 1 million regions.
+	// It takes about 1.3 minutes to iterate 1 million regions(with DefaultPatrolRegionInterval=10ms).
```
Comment (lhy1024) [inline pkg/schedule/coordinator.go:55]: This is an ideal situation, if the consumer side is smaller than the production side, then it will take longer

Comment (AndreMouche) [reply pkg/schedule/coordinator.go:55]: yes

Comment (HuSharp) [reply pkg/schedule/coordinator.go:55]: Is it better to write `1000000/128*10/60/1000` here to indicate the calculation progress and emphasize it's an ideal situation?

**Review by nolouch — APPROVED** [review]

**Pull request merged**